addons/spreadsheet/static/src/pivot/plugins/pivot_core_plugin.js

```diff
@@ -195,7 +193,7 @@ export class PivotCorePlugin extends OdooCorePlugin {
      *
      * @returns {PivotDefinition}
      */
-    getPivotDefinition(id) {
+    getPivotRawDefinition(id) {
```

Comment:
I'm not sure about the "raw" here.
I think I would either just leave it `getPivotDefinition` (and rename `getPivotRuntimeDefinition` to `getPivotRuntime`? In o-spreadsheet, we have `getChartRuntime`, not `getChartRuntimeDefinition`).
Or maybe `getPivotCoreDefinition`?